### PR TITLE
Add sample code

### PR DESCRIPTION
**Pull request by andrewkim316**
Adds a mean-reversion paper-trading bot to the Javascipt SDK.  Translated directly from the C# SDK mean-reversion script.

### CODE REVIEW

File: examples/mean-reversion.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      orders = resp;
+    });
+    orders.forEach(async (order) => {
+      this.alpaca.cancelOrder(order.id).catch((e) => {});
+    });
+
+    // Get the running average of prices of the last 20 minutes
+    var bars;
+    await this.alpaca.getBars('minute',this.stock).then((resp) => {
+      bars = resp.AAPL;
```
Comment (umitanuki) [inline examples/mean-reversion.js]: Should this be `this.stock`?

File: examples/mean-reversion.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    console.log("Market Opened.");
+
+    // Figure out when the market will close so we can prepare to sell beforehand.
+    var closingTime;
+    var currTime;
+    await this.alpaca.getClock().then((resp) =>{
+      closingTime = new Date(resp.next_close.substring(0,resp.next_close.length-6));
+      currTime = new Date(resp.timestamp.substring(0,resp.timestamp.length-6));
+    });
+    this.timeToClose = Math.abs(closingTime - currTime);
```
Comment (ttt733) [inline examples/mean-reversion.js]: Shouldn't need an abs here, right?

**Comment by ttt733** [comment]
It looks like it will start executing buys and sells immediately on market open rather than waiting for 20 minutes of data first. Wouldn't the overnight jump mess up the running average? Seems to me like it should start executing 20 minutes after open.

File: examples/mean-reversion.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    var positionQuantity = 0;
+    var positionValue = 0;
+
+    // Get our position, if any
+    try{
+      await this.alpaca.getPosition(this.stock).then((resp) => {
+        positionQuantity = resp.qty;
+        positionValue = resp.market_value;
+      });
+    } catch (err){/*console.log(err);*/}
```
Comment (ttt733) [inline examples/mean-reversion.js]: Should probably log these errors.

**Pull request merged**